README.md

```diff
@@ -1,6 +1,6 @@
-# Bing Ads (Source)
+# Microsoft Advertising (Source)
```

Comment:
I know the name of the repo will be `dbt_microsoft_ads` for brevity's sake, but I think we should use the correct name whenever possible.